### DISCUSSION TITLE
Compile error

### DISCUSSION THREAD

**Discussion by baileys-am** [chakra-ui/chakra-ui#5068] [Q&A] [answered]
Hi this is my first time trying to use chakra-ui. I'm receiving the following error when app is loaded via react-script start. I'm using a React ASP.NET typescript app created via `dotnet new`.

```
./node_modules/@chakra-ui/menu/dist/chakra-ui-menu.esm.js 103:15
Module parse failed: Unexpected token (103:15)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
|   }, [descendants]);
|   const onOpenInternal = React.useCallback(() => {
>     onOpenProp?.();
| 
|     if (autoSelect) {
```

Package versions:
    "@chakra-ui/react": "^1.7.0",
    "@emotion/react": "^11.5.0",
    "@emotion/styled": "^11.3.0",
    "framer-motion": "^4.1.17",
    "react-scripts": "4.0.3",
    "typescript": "^4.4.4"

Node version : 16.13.0
npm version: 8.13.

**Comment by primos63** [comment, marked as answer]
In `package.json` for your app, replace the `browserlist` with
```
  "browserslist": [
    ">0.2%",
    "not dead",
    "not op_mini all"
  ]
```
and 1.7.0 should work.  Make sure you delete your `.cache` file in the `node_modules` directory.

**Update:**
The issue is due to the nullish coalescing operator (??) and react-scripts needing to be updated to handle it. 
[Issue](https://github.com/facebook/create-react-app/issues/9468#issuecomment-952147361) detailed in Facebook's CRA repo.

**Reply by indexhui** [reply]
THX! It works for me ~~

**Reply by baileys-am** [reply]
That fixed it. Thanks!

**Reply by jjeffrin** [reply]
![image](https://user-images.githubusercontent.com/25849066/141492933-a0565658-89bd-4e38-9e23-6ee0e9c326ef.png)

Nah, this isn't helping me. Am I doing this correctly ?

Also, tried the following:
![image](https://user-images.githubusercontent.com/25849066/141493170-748da98e-85e0-4eeb-a5e5-0c3d8e94ebe2.png)

**Reply by trungthecelestial** [reply]
same as @jjeffrin, not able to build even when change current browserslist

**Reply by primos63** [reply]
Did you delete the .cache file?

**Reply by jjeffrin** [reply]
You mean the .cache folder? Yes i deleted it. Still no luck.

**Reply by primos63** [reply]
Interesting. Wonder why it works for some and not others. If it's not working what did you use to create the app? CRA, Next, something else?

Note: A PR, #5075, was submitted today to fix the issue.

**Reply by jjeffrin** [reply]
CRA.

**Reply by trungthecelestial** [reply]
It's blocking my company plan to release today. How long will the PR is published ? Is there any other way I can fixed that. I noticed that our build yesterday is still fine, But today it is blocked by the above error. Btw, I used react-boilerplate.

**Reply by HikaruFN** [reply]
getting the same problem here!

**Reply by primos63** [reply]
@trungthecelestial I'm sorry this issue is holding up your release. This is an issue that has not only affected Chakra, but also other React libraries. The issue is documented on [Facebook's CRA repo](https://github.com/facebook/create-react-app/issues/9468#issuecomment-952147361). You could try the CRACO method, but not sure you want to introduce that right before a release. I do not have a timeline. I know from notifications that the PR was just pushed. As for alternatives, I'd have to create a project with react-boilerplate to see what it's using (cra scripts?).

@jjeffrin That should work in your case. If you have your project in a GH repo that I can access, I'd be happy to give it a try.

**Reply by primos63** [reply]
Chakra v1.7.1 has been released!!

**Reply by jjeffrin** [reply]
Thanks for following this up! I can confirm this issue is not reproducible anymore! 👍
